### DISCUSSION TITLE
Задержка дыхания

### DISCUSSION THREAD

**Discussion by Unknow123456789** [octothorp-team/public-dobrograd#849] [Предложения]
Привет! Хочу предложить задержку дыхании при прицеливании, а то есть при нацеливании с оружия можно будет временно нормально целиться что бы прицел не дергался, но это будет на время.

**Comment by Hardcore-punk** [comment]
Смысла от этого не особо много, при стрельбе будет все та же тряска, а когда целишься долго не держи просто так пкм, я тоже до бана еще замечал, что пару секунд прицел ровно, а потом ведет
